CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- Return 404 back if the upstream is not defined [THREESCALE-3775](https://issues.jboss.org/browse/THREESCALE-3775) [PR #1129](https://github.com/3scale/APIcast/pull/1129)
+
+## [Unreleased]
```

Comment:
This looks wrong. There are 2 "unreleased" sections.

Reply:
yes, this is to avoid merge conflicts with 3.7 release.